srcalc/src/main/java/gov/va/med/srcalc/web/DynamicValueVisitor.java

```diff
@@ -12,6 +12,10 @@
 {
     private VariableEntry fVariableEntry;
     
+    /**
+     * Constructs an instance.
```

Comment:
I keep on forgetting what `DynamicValueVisitor` actually does. The name is not informative to me. Can we think of a better one? What about `ValuePopulatorVisitor`?